<launch>
   <arg name="sim_topic" default="/sensors/dvl"/>
   <arg name="real_topic" default="/dvl/dvl"/>

   <node name="nortek_dvl" pkg="riptide_gazebo" type="nortek_dvl_node" output="screen">
      <param name="sim_topic" value="$(arg sim_topic)"/>
      <param name="real_topic" value="$(arg real_topic)"/>
   </node>

</launch>